dlc_afterhours/terbyte.lua

```diff
@@ -0,0 +1,141 @@
+-- After Hours Terbyte Interior: -1421.015, -3012.587, -80.000
```

Comment:
[Terrorbyte](https://gta.fandom.com/wiki/Terrorbyte)

Reply:
fixed in c65fbe4 and 6c38931